Cargo.toml

```diff
@@ -225,3 +225,13 @@ VE = []
 VN = []
 ZM = []
 ZW = []
+
+[lints.rust]
+unsafe_code = "forbid"
+
+[lints.clippy]
+enum_glob_use = "forbid"
+pedantic = { level = "deny", priority = -1 }
+missing_errors_doc = "allow"
+too_many_lines = "allow"
+wildcard_imports = "allow"
```

Comment:
These are for the generated files? Better instead to add clippy attributes to those files only?

Reply:
Some were in the normal code files, but i applied `#[allow(...)]` in the relevant places